egs/tedlium3/ASR/conformer_ctc2/conformer.py

```diff
@@ -0,0 +1,1069 @@
+#!/usr/bin/env python3
```

Comment:
Is it possible to replace this file with a symlink to the one from librispeech's `conformer_ctc2`?

Reply:
@csukuangfj Not sure really, as I mentioned in the first comment to this PR, I've added `RandomCombine` and `aux_layers` to `ConformerEncoder`, not sure that this is compatible with librispeech `conformer_ctc2`